float64 x
float64 y
float64 z
float64 Vel2D
